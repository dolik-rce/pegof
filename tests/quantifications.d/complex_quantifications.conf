input quantifications.d/complex_quantifications.peg
optimize double-quantification
header never
