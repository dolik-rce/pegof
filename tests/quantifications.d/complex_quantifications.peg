X <-
    (("A" . [bB]*)?)?
    / (("A" . [bB]* / "C")?)?
