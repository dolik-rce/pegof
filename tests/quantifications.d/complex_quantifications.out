X <-
    ("A" . [\s]*)?
    / ("A" . [\s]* / "B")?
